package.json

```diff
@@ -42,8 +42,7 @@
     "@types/node": "^20.0.0",
     "axios": "^1.0.0",
     "body-parser": "^1.20.0",
-    "file-type": "^16.5.4",
-    "form-data": "^4.0.0"
```

Comment:
Note: removed the 'form-data'. Because this pull-request make to use the NodeJS' native `FormData` class instead.